packages/server/src/koaJsonRpc/index.ts

```diff
@@ -188,37 +165,19 @@ export default class KoaJsonRpc {
 
   async getRequestResult(request: IJsonRpcRequest, ip: string): Promise<IJsonRpcResponse> {
```

Comment:
I think we should `unknown` here given `request` has not been validate it yet

```suggestion
  async getRequestResult(request: unknown, ip: string): Promise<IJsonRpcResponse> {
```

We should change its type after `validateJsonRpcRequest`.

Reply:
This was already defined as IJsonRpcRequest from the top in the rpcApp() method when parsed from the request context object. Why would we wanna change it to unknown? If it's the case, the entire pipeline would need to be updated from the top, which is beyond the scope of this PR.